tests/jballerina-unit-test/src/test/resources/test-src/statements/variabledef/configurable_var_decl.bal

```diff
@@ -25,9 +25,22 @@ configurable UserInfo admin = {
     password: "password"
 };
 
+public type CustomConfiguration record {
+    string username;
+    string logLevel = "OFF";
+};
+
+configurable CustomConfiguration customConfig1 = {username: "chiranS"};
+
+configurable CustomConfiguration customConfig2 = {username: "chiranS", logLevel: "DEBUG"};
+
```

Comment:
Please add a test when the entire value is provided too. Maybe with ?.

Reply:
What you mean by entire value? Small example?

Reply:
configurable CustomConfiguration customConfig3 = ?;